src/static/scope.jl

```diff
@@ -0,0 +1,107 @@
+#=
+Scope information for `EXRR`.
+
+NOTE:
+- StaticLint.jl adds various scope-related information for `EXPR.meta.scope` field,
+    but we currently just need to know whether `EXPR` introduces a scope or not.
+    So let's just make a function to check that, and don't fill `EXPR` with scope information.
+- Adapted from https://github.com/julia-vscode/StaticLint.jl/blob/cd8935a138caf18385c46db977b52c5ac9e90809/src/scope.jl
+=#
+
+function hasscope(x::EXPR)
+    t = typof(x)
+
+    # NOTE: added conditions below when adapted
+    if t === CSTParser.TupleH && (p = parentof(x)) !== nothing && !hasscope(p)
+        return true
+    # # XXX:
+    # # introduced in https://github.com/JunoLab/Atom.jl/commit/7de5299001395b83bab0cb9d102e3f36d5c202d1
+    # # but now this seems to include bad cases for function arguments
+    # elseif t === CSTParser.Call && (p = parentof(x)) !== nothing && !hasscope(p)
+    #     return true
```

Comment:
@pfitzseb 
This scope condition check is introduced in https://github.com/JunoLab/Atom.jl/commit/7de5299001395b83bab0cb9d102e3f36d5c202d1, but now it causes rare case problems like #223 .
This is because `CSTParser.Call` also represents function arguments and if a `LocalScope` is introduced for it, `actual_localbindings` filters it out because of [`span` check](https://github.com/JunoLab/Atom.jl/blob/5962641e3fc61a32d69155676dee0dd6585589ae/src/static/local.jl#L131-L133), in some cases.
Ideally we want to make a span of function argument `Call` equivalent to the span of its `FunctionDef`, but I'm not sure it's really possible, since I think we can't tell the difference between function argument `Call` and usual `Call`s in function body.

If we don't introduce `LocalScope` for it, then function arguments are not filtered out.

***

So could you tell me why you wanted to introduce scopes for `Call`s ? I guess we actually don't need this and tests are passed if we don't have this check.

Reply:
This might've been to keep kwargs in function calls out of the local completions/outline. As you said, this doesn't seem to be necessary anymore, so we can in fact remove this check.